pyproject.toml

```
@@ -1,16 +1,18 @@
 [build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
 
-[tool.poetry]
+[project]
 name = "ruff-lsp"
-version = "0.0.9"
-description = "A Language Server Protocol implementation for Ruff."
-authors = ["Charlie Marsh <charlie.r.marsh@gmail.com>"]
-maintainers = ["Charlie Marsh <charlie.r.marsh@gmail.com>"]
-license = "MIT"
-readme = "README.md"
-repository = "https://github.com/charliermarsh/ruff-lsp"
+dynamic = ["version", "description"]
+authors = [
+    { name = "Charlie Marsh", email = "charlie.r.marsh@gmail.com" },
+]
+maintainers = [
+    { name = "Charlie Marsh", email = "charlie.r.marsh@gmail.com" },
+]
+requires-python = ">=3.7"
+license = { file = "LICENSE" }
```

https://peps.python.org/pep-0639/

```suggestion
license = "MIT"
```